reportportal_client/service.py

```diff
@@ -244,6 +253,17 @@ def finish_test_item(self,
                          issue=None,
                          attributes=None,
                          **kwargs):
+        """
```

Comment:
```suggestion
        """ Finish the test item and return HTTP response.
       
        :param item_id:    id of the test item
        :param end_time:   time in UTC format
        :param status:     status of the test
        :param issue:      description of an issue
        :param attributes: list of attributes
        :param kwargs:     other parameters
        :return:           json message
        """
```

Please update similar doc strings the same way.

Reply:
Still actual...